ResourceStatusSystem/PolicySystem/Actions/SlackAction.py

```diff
@@ -0,0 +1,122 @@
+""" SlackAction
+
+  This action posts the status change to a Slack channel
+  It uses Slack Webhooks.
+  Also compatible with Mattermost, which is an open source alternative to Slack.
+
+  To create a webhook URL refer the following :
```

Comment:
This block will just render as one paragraph.

Reply:
Corrected.